doc/api/buffer.md

```diff
@@ -330,7 +330,7 @@ changes:
 
 > Stability: 0 - Deprecated: Use [`Buffer.from(array)`] instead.
 
-* `array` {Array} An array of bytes to copy from
+* `array` {Integer[]} An array of bytes to copy from.
```

Comment:
Nit - I think the integer should be lower cased even if it is an array.

Reply:
Done.